Name: Hide XTerm Icon
Version: 2.2.4-1
Author: webOS Archive
Description: Hide XTerm Icon from Launcher -- but keep XTerm installed

--- .orig/media/cryptofs/apps/usr/palm/applications/org.webosinternals.xterm/appinfo.json
+++ /media/cryptofs/apps/usr/palm/applications/org.webosinternals.xterm/appinfo.json
@@ -6,6 +6,8 @@
 "type": "web",
 "main": "index.html",
 "icon": "icon.png",
+"removable": true,
+"visible": false,
 "message": "XTerm is used to provide a Linux command-line interface on webOS.<br> Due to webOS security constraints, this application cannot run directly.<br> You must use <b>Xecutah</b> to run the XTerm and XServer applications on webOS.",
 "changeLog": [ { "version" : "0.9.7",  "log" : [ "UTF8 character support, set metaSendsEscape for compat." ] }, { "version" : "0.9.5",  "log" : [ "Preserve the environment (including the DISPLAY variable) inside the terminal login" ] }, { "version" : "0.9.3",  "log" : [ "Added support for starting multiple applications in separate cards" ] }, { "version" : "0.9.2",  "log" : [ "Move to an event based start" ] }, { "version" : "0.9.1",  "log" : [ "Added multi-card support" ] }, { "version" : "0.9.0",  "log" : [ "First public feed release" ] }, { "version" : "0.7.7",  "log" : [ "Set default colors to green-on-black, not black-on-black" ] }, { "version" : "0.7.6",  "log" : [ "Add ability to change xterm colors via tweaks (thanks to rrix!)" ] }, { "version" : "0.7.2",  "log" : [ "New upstart-based execution architecture to support legacy webOS versions" ] }, { "version" : "0.6.6",  "log" : [ "Initial testing release" ] } ],
 "uiRevision": "2",
